docs/content/3.manual/manual.yml

```diff
@@ -271,17 +271,24 @@ sections:
             input: '"Hello, world!"'
             output: ['"Hello, world!"']
 
-      - title: "`.foo`, `.foo.bar`"
+      - title: "Object Identifier-Index: `.foo`, `.foo.bar`"
```

Comment:
I think `Member Selector-Index:` might be clearer, since `.foo` doesn't return an object, what it identifies is a *key*, and it uses syntax that most people will recognize as "dot member" notation.